apps/src/code-studio/pd/application/teacher1819/Section4SummerWorkshop.jsx

```diff
@@ -183,8 +184,8 @@ export default class Section4SummerWorkshop extends ApplicationFormComponent {
         return (
           <div>
             <h5>
-              Your region’s assigned summer workshop will be
-              {' ' + this.state.partnerWorkshops[0].dates + ' '} in
+                Your region’s assigned summer workshop will be
```

Comment:
Intentional? Are you trying to indent this paragraph?

Reply:
unintentional. will fix